server/config.json

```diff
@@ -21,5 +20,8 @@
     "pgSslCert": "./server/ssl/server.crt",
     "secret": "Your own session key here",
     "sessionTimeout": 86400000,
+    "features": {
```

Comment:
Let's not add new objects in for options. As it is, I want to get npmModules out of here, because eventually I want to be able to handle all these options as environment variables at run time, but objects don't work for that.